app/build.gradle

```diff
@@ -137,8 +137,16 @@ dependencies {
     implementation "androidx.navigation:navigation-fragment-ktx:2.5.3"
     implementation "androidx.navigation:navigation-ui-ktx:2.5.3"
 
+
+    implementation "androidx.recyclerview:recyclerview-selection:1.1.0"
+    implementation 'androidx.core:core-ktx:1.6.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.3.0'
+
     // https://developers.google.com/nearby
     implementation 'com.google.android.gms:play-services-nearby:18.5.0'
+    implementation 'androidx.test:core-ktx:1.4.0'
```

Comment:
It's a testing library. `implementation` means we will add that library everywhere. For libs required for testing you should use `testImplementation`. That ensures that the dependency is not included in the application itself, but available when running tests.
But as you haven't written tests in this merge request this is an unnecessary chanmge. Please remove